debugtools/DDR_VM/src/com/ibm/j9ddr/AuxFieldInfo29.dat

```diff
@@ -564,6 +564,15 @@ J9ThreadMonitorPool.entries = required
 J9ThreadMonitorPool.next = required
 J9TranslationBufferSet.romClassBuilder = required
 J9UTF8.length = required
+J9VMContinuation.arg0EA = UDATA*
+J9VMContinuation.decompilationStack = J9JITDecompilationInfo*
+J9VMContinuation.i2jState = J9I2JState
+J9VMContinuation.j2iFrame = UDATA*
+J9VMContinuation.jitGPRs = UDATA*
```

Comment:
This must use the actual type:
```
J9VMContinuation.jitGPRs = J9JITGPRSpillArea
```
otherwise, accessing a core file produced by
```
IBM Semeru Runtime Open Edition 20.0.1.0 (build 20.0.1+9)
Eclipse OpenJ9 VM 20.0.1.0 (build openj9-0.39.0, JRE 20 Linux amd64-64-Bit Compressed References 20230418_50 (JIT enabled, AOT enabled)
```
fails. That version has continuations (the class `J9VMContinuationPointer` is present, but the method `jitGPRs()` wouldn't have the expected signature), leading to this stack trace:
```
> !continuationstack 0x00000000FFE47E48
Error executing DDR command !continuationstack 0x00000000FFE47E48 : null
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at openj9.dtfjview/com.ibm.jvm.dtfjview.CombinedContext.executeDDRInteractiveCommand(CombinedContext.java:258)
	at openj9.dtfjview/com.ibm.jvm.dtfjview.CombinedContext.execute(CombinedContext.java:169)
	at openj9.dtfjview/com.ibm.jvm.dtfjview.Session.execute(Session.java:813)
	at openj9.dtfjview/com.ibm.jvm.dtfjview.Session.execute(Session.java:767)
	at openj9.dtfjview/com.ibm.jvm.dtfjview.tools.ToolsRegistry.executeJdmpviewCommand(ToolsRegistry.java:183)
	at openj9.dtfjview/com.ibm.jvm.dtfjview.tools.ToolsRegistry.execute(ToolsRegistry.java:107)
	at openj9.dtfjview/com.ibm.jvm.dtfjview.tools.ToolsRegistry.execute(ToolsRegistry.java:88)
	at openj9.dtfjview/com.ibm.jvm.dtfjview.tools.ToolsRegistry.recordAndExecute(ToolsRegistry.java:72)
	at openj9.dtfjview/com.ibm.jvm.dtfjview.Session.runInteractive(Session.java:741)
	at openj9.dtfjview/com.ibm.jvm.dtfjview.Session.run(Session.java:665)
	at openj9.dtfjview/com.ibm.jvm.dtfjview.DTFJView.launch(DTFJView.java:52)
	at openj9.dtfjview/com.ibm.jvm.dtfjview.DTFJView.main(DTFJView.java:46)
Caused by: java.lang.NoSuchMethodError: com/ibm/j9ddr/vm29/pointer/generated/J9VMContinuationPointer.jitGPRs()Lcom/ibm/j9ddr/vm29/pointer/UDATAPointer; (loaded from <Unknown> by J9DDRClassloader for com.ibm.j9ddr.vm29.) called from class com.ibm.j9ddr.vm29.j9.stackwalker.StackWalker$StackWalker_29_V0 (loaded from <Unknown> by J9DDRClassloader for com.ibm.j9ddr.vm29.).
	at com.ibm.j9ddr.vm29.j9.stackwalker.StackWalker$StackWalker_29_V0.walkStackFrames(StackWalker.java:217)
	at com.ibm.j9ddr.vm29.j9.stackwalker.StackWalker.walkStackFrames(StackWalker.java:109)
	at com.ibm.j9ddr.vm29.tools.ddrinteractive.commands.ContinuationStackWalkCommand.run(ContinuationStackWalkCommand.java:92)
	at openj9.dtfj/com.ibm.j9ddr.tools.ddrinteractive.Context.tryCommand(Context.java:229)
	at openj9.dtfj/com.ibm.j9ddr.tools.ddrinteractive.Context.execute(Context.java:202)
	at openj9.dtfj/com.ibm.j9ddr.tools.ddrinteractive.DDRInteractive.execute(DDRInteractive.java:356)
	at openj9.dtfj/com.ibm.j9ddr.command.CommandReader.processLine(CommandReader.java:78)
	at openj9.dtfj/com.ibm.j9ddr.tools.ddrinteractive.DDRInteractive.processLine(DDRInteractive.java:331)
	... 16 more
```
In order to use that type, it must be made known to DDR (in older versions) by adding this in `vmddr.cpp` (after `J9JITFrame`):
```
VM_DdrDebugLink(J9JITGPRSpillArea)
```